dev/io.openliberty.cdi.4.1.internal.web/bnd.bnd

```diff
@@ -0,0 +1,28 @@
+#*******************************************************************************
+# Copyright (c) 2024 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-Name: io.openliberty.cdi.4.1.internal.web
+Bundle-SymbolicName: io.openliberty.cdi.4.1.internal.web
+Bundle-Description: CDI 4.1 Web, version ${bVersion}
+
```

Comment:
```suggestion

javac.source: 17
javac.target: 17

Require-Capability: osgi.ee; filter:="(&(osgi.ee=JavaSE)(version=17))"
```

Reply:
Thanks, I missed this as it wasn't in the original bnd.bnd for the 4.0 bundle! I'll update the 4.1 bundle.